srcpkgs/volta/template

```diff
@@ -0,0 +1,77 @@
+# Template file for 'volta'
+pkgname=volta
+version=0.7.1
+revision=2
+archs="x86_64 i686"
+build_helper="rust"
+hostmakedepends="cargo pkg-config"
+makedepends="libressl-devel"
+short_desc="JavaScript tool version sync helper"
+maintainer="Alex Lohr <alex.lohr@logmein.com>"
+license="BSD-2-Clause"
+homepage="https://volta.sh/"
+distfiles="https://github.com/volta-cli/${pkgname}/archive/v${version}.tar.gz"
+checksum=e53a07e167bb64103f36901423f5a377a2ea89ecfdd7a1343e69d659f99f9c1b
+
+post_patch() {
+	cat >> shell/unix/install.sh <<_EOF
```

Comment:
This should go in $FILESDIR and be installed from there, rather than being inlined here as a massive heredoc.

Reply:
fixed